static_content/contest_rules.html

```diff
@@ -0,0 +1,69 @@
+<div>
+  You will be asked to confirm you followed each rule upon submission. Repeated rule violators will
+  be banned from the contest for the rest of the year.
```

Comment:
So if you break the rules in January you are banned 11 months, but December only the 1?

Reply:
That is apparently what the rules say: https://imgur.com/lm0yPy7. Maybe something to bring up with VertigoOne?